src/course-home/progress-tab/certificate-status/CertificateStatus.jsx

```diff
@@ -42,13 +46,16 @@ const CertificateStatus = ({ intl }) => {
     certificateAvailableDate,
   } = certificateData || {};
 
+  const entranceExamPassed = entranceExamData ? entranceExamData.entranceExamPassed : null;
```

Comment:
Nit - you can write this more concisely as:
```suggestion
  const entranceExamPassed = entranceExamData?.entranceExamPassed ?? null;
```

Reply:
Yes, thanks for the suggestion. Have made the changes.